saleor/settings.py

```diff
@@ -525,13 +530,12 @@ def get_host():
 # This enable the new 'No Captcha reCaptcha' version (the simple checkbox)
 # instead of the old (deprecated) one. For more information see:
 #   https://github.com/praekelt/django-recaptcha/blob/34af16ba1e/README.rst
-NOCAPTCHA = True
+NOCAPTCHA = get_bool_from_env('NOCAPTCHA', True)
```

Comment:
Could you change the comment to say reCaptcha (v1) was shutdown instead of deprecated?
(And restore the `True` non-changeable constant as explained in my above comment).

Reply:
:+1: will do